src/CollapsibleGroup/index.js

```diff
@@ -108,15 +116,13 @@ CollapsibleGroup.defaultProps = {
 CollapsibleGroup.propTypes = {
   className: PropTypes.string,
   allowMultiple: PropTypes.bool,
-  collapsibles: PropTypes.arrayOf(
-    PropTypes.shape({
-      ...CollapsiblePropTypes,
-      children: PropTypes.node,
-    }),
-  ).isRequired,
   transTime: PropTypes.number,
   transCurve: PropTypes.string,
   classPrefix: PropTypes.string,
+  children: PropTypes.oneOfType([
```

Comment:
The definition for `children` here can just be `PropTypes.node`, which qualifies both an array of nodes _and_ single nodes.